receiver/lokireceiver/metadata.yaml

```diff
@@ -9,3 +9,6 @@ status:
   - sumo
   codeowners:
     active: [mar4uk, jpkrohling]
+
+tests:
+  config:
```

Comment:
Unrelated to this PR. Once we apply this all components, we will enable the test generation by default, and these line won't be needed anymore right?

Reply:
That sounds like a good next step to me.